### PR TITLE
OBSDOCS-197: Clarify information about secrets in custom Alertmanager…

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.12` and later

Issue: [OBSDOCS-197](https://issues.redhat.com/browse/OBSDOCS-197)

Links to docs preview: 
* [Preparing to configure the monitoring stack](https://73723--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/configuring-the-monitoring-stack#preparing-to-configure-the-monitoring-stack)
* [Support considerations for monitoring](https://73723--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/configuring-the-monitoring-stack#support-considerations_configuring-the-monitoring-stack)

QE review:
- [x] QE has approved this change.

**Additional information:**

### CODE REVIEW

File: modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [id="creating-user-defined-workload-monitoring-configmap_{context}"]
 = Creating a user-defined workload monitoring config map
 
-To configure the components that monitor user-defined projects, you must create the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project.
+You can configure the Cluster Monitoring Operator (CMO) by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures the components that monitor user-defined projects.
```
Comment (juzhao) [inline modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc:9]:
```suggestion
You can configure the user workload monitoring components by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project. 
```

File: modules/monitoring-creating-cluster-monitoring-configmap.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [id="creating-cluster-monitoring-configmap_{context}"]
 = Creating a cluster monitoring config map
 
-To configure core {product-title} monitoring components, you must create the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+You can configure the Cluster Monitoring Operator (CMO) by creating the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures core {product-title} monitoring components.
```
Comment (juzhao) [inline modules/monitoring-creating-cluster-monitoring-configmap.adoc:9]:
```suggestion
You can configure the core {product-title} monitoring components by creating the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project. The CMO then configures core {product-title} monitoring components.
```
we configured core monitoring components, example: alertmanager/prometheus, not for CMO

File: modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc (hunk 1 above)
Comment (eromanova97) [reply modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc:9]: @juzhao 
Should we not specify here that it is also done by the CMO?
```suggestion
You can configure the user workload monitoring components by creating the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project. The Cluster Monitoring Operator (CMO) then configures the components that monitor user-defined projects.
```

Comment (eromanova97) [reply modules/monitoring-creating-user-defined-workload-monitoring-configmap.adoc:9]: Just saw you edited comment :smile: :+1:

**Review by JoeAldinger — COMMENTED** [review]
lgtm

**Pull request merged**